xblockmufi/private/view.less

```diff
@@ -1,3 +1,19 @@
-.xblockmufi_block {
-    // TODO: Put your styles here
+.expert_answer {
```

Comment:
The benefit to nesting all the selectors inside `xblockmufi_block` is that you can more easily target your CSS selectors and not risk accidentally styling elements elsewhere on the page (like any element with an `explanation` class).

You may want to restore this..